framework/console/controllers/AssetController.php

```diff
@@ -326,6 +327,11 @@ protected function loadTargets($targets, $bundles)
     protected function buildTarget($target, $type, $bundles)
     {
         $tempFile = $target->basePath . '/' . strtr($target->$type, ['{hash}' => 'temp']);
+        if (!is_file($tempFile)) {
+            FileHelper::createDirectory(dirname($tempFile), 0755, true);
+            touch($tempFile);
```

Comment:
I can't unserstand how `touch()` relates to this: `$tempFile` will be passed to `file_put_contents()` in the end.